webui/src/context/TableSettingsContext.tsx

```diff
@@ -1,41 +1,55 @@
-// src/context/TableSettingsContext.tsx
 import React, { useState, useContext, useEffect, ReactNode } from "react";
 
+// Combined table settings interface
 export interface TableSettings {
-  showNodeId: boolean;
-  showName: boolean;
-  showType: boolean;
-  showEnv: boolean;
-  showInputs: boolean;
-  showOutputs: boolean;
-  showVersion: boolean;
-  showAction: boolean;
+  // Jobs Table
+  showJobId?: boolean;
+  showJobName?: boolean;
+  showCreated?: boolean;
+  showProgram?: boolean;
+  showJobType?: boolean;
+  showLabel?: boolean;
+  showStatus?: boolean;
+  // Nodes Table
+  showNodeId?: boolean;
+  showNodeName?: boolean;
+  showNodeType?: boolean;
+  showEnv?: boolean;
+  showInputs?: boolean;
+  showOutputs?: boolean;
+  showVersion?: boolean;
+  showAction?: boolean;
```

Comment:
We added customisable columns so that we could get more node details available in the table. Currently we're missing labels and capacity. Could you (add a new ticket to) add those in plz?